packages/js/src/elementor/components/fills/ElementorFill.js

```diff
@@ -11,8 +11,8 @@ import InsightsModal from "../../../insights/components/insights-modal";
 import Alert from "../../containers/Alert";
 import { KeywordInput, ReadabilityAnalysis, SeoAnalysis, InclusiveLanguageAnalysis } from "@yoast/externals/components";
 import SidebarItem from "../../../components/SidebarItem";
-import SearchAppearanceModal from "../modals/editorModals/SearchAppearanceModal";
-import SocialAppearanceModal from "../modals/editorModals/SocialAppearanceModal";
+import SearchAppearanceModal from "../../../components/modals/editorModals/SearchAppearanceModal";
+import SocialAppearanceModal from "../../../components/modals/editorModals/SocialAppearanceModal";
```

Comment:
I've unified the modals used in the sidebar for the block editor and Elementor as they were exactly the same.